stories/Organism/Statscardslider/statscardslider.scss

```diff
@@ -2,7 +2,25 @@
 @import '../../assets/scss/variables';
 
 .stats-slider {
-  .stats-card {
+  .stats-card.small {
     max-width: 100%;
   }
+
+  .swiper-scrollbar {
+    @include devicebreak(medium) {
+      padding-right: $spacing-44;
+    }
+  }
 }
+
+[dir='rtl'] {
+  .stats-slider {
+    &.swiper-initialized {
+      .swiper-scrollbar {
+        @include devicebreak(medium) {
+          padding-left: $spacing-44;
```

Comment:
@Shamimkhan321 We need to set the `padding-right: 0` in the above as well as we need this both on `.swiper-initialized` and `.swiper-scrollbar` right?